src/plugins/discover/public/hooks/use_navigation_props.tsx

```diff
@@ -41,7 +41,7 @@ const getStateParams = ({
   let appliedFilters: Filter[] = [];
   let query: Query | AggregateQuery | undefined;
   let timeRange: TimeRange | undefined;
-  if (!isEmbeddableView) {
+  if (!isEmbeddableView && filterManager) {
```

Comment:
ℹ️ Making this optional, as we don't allow the creation/edit of DSL filters in ES|QL tables (we just append to the ES|QL query)